corehq/apps/locations/fixtures.py

```diff
@@ -28,20 +29,20 @@ def __contains__(self, item):
         return item in self.by_id
 
 
-def fixture_last_modified(user):
+def fixture_last_modified(restore_user):
     """Return when the fixture was last modified"""
-    return user.fixture_status(UserFixtureType.LOCATION)
+    return restore_user.fixture_status(UserFixtureType.LOCATION)
```

Comment:
maybe addressed in a later commit, but this is the type of thing I would have thought could be converted to a function instead of being defined on the class